modules/csharp/pages/gs-install.adoc

```diff
@@ -56,13 +56,13 @@ This is the recommended method of dependency management because it supports the
 
 .`Couchbase.Lite.Enterprise`
 * `Couchbase.Lite.Enterprise.Support.ios`
-* `Couchbase.Lite.Enterprise.Support.android`
+* `Couchbase.Lite.Enterprise.Support.Droid`
 * `Couchbase.Lite.Enterprise.Support.NetDesktop`
 * `Couchbase.Lite.Enterprise.Support.WinUI`
 
 .`Couchbase.Lite`
 * `Couchbase.Lite.Support.ios`
-* `Couchbase.Lite.Support.android`
+* `Couchbase.Lite.Support.Droid`
```

Comment:
This is not a namespace but a package name so this is correct as Android (upper case)

Reply:
Done!